Makefile

```diff
@@ -0,0 +1,39 @@
+.PHONY: all
+all: bin/openshift-install check ## Build and check everything
+
+.PHONY: check
+check: fmt vet lint test ## Check your code and run tests
+
+.PHONY: revendor
+revendor: ## Revendor go packages
+	rm glide.lock
+	glide install --strip-vendor
+	glide-vc --use-lock-file --no-tests --only-code
+
+.PHONY: bin/terraform 
+bin/terraform: ## Get terraform package
+	hack/get-terraform.sh
+
+.PHONY: bin/openshift-install
+bin/openshift-install: ## build installer binary
+	hack/build.sh
+
+.PHONY: test
+test: ## Run unit test
```

Comment:
Nit: "nit test" -> "unit tests" (we have several ;).  Also, maybe mention Go ("Go unit tests").